internal/tweets/edit.go

```diff
@@ -44,7 +45,7 @@ func (v *Visibility) isValid() bool {
 
 func (s *Service) Edit(w http.ResponseWriter, r *http.Request) {
 	tweetID := mux.Vars(r)["id_tweet"]
-	userID := r.Context().Value("userID").(int)
+	userID := r.Context().Value("userID").(string)
```

Comment:
this will break once https://github.com/mo-pm888/twitter/pull/17/files#diff-37ac464c45e06bed5132281126e435b869de3ca34bbdfb008db79edad30c8edcR43 is merged